scripts/xapi.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+SourcePath=@LIBEXECDIR@/xapi-init
+Description=XenServer server (XAPI)
+After=network-online.target syslog.target forkexecd.service xcp-networkd.service message-switch.service v6d.service save-boot-info.service attach-static-vdis.service genptoken.service xcp-rrdd.service xapi-storage-script.service xenopsd-xc.service xenstored.service
+Conflicts=shutdown.target
+
+[Service]
+Type=notify
+Restart=on-failure
+TimeoutSec=5min
+IgnoreSIGPIPE=no
+KillMode=process
```

Comment:
So this will be changed to `Type=simple` and I guess we'll keep `Restart=on-failure`, but the other three line can go away?